lib/EditableList/readme.md

```diff
@@ -53,6 +53,11 @@ fieldComponents | object | Allows custom components for edit mode to be used. Fi
 columnWidths | object | Allows custom column widths to be set. If you use this, be sure to set a width for an 'actions' column as part of this object. | | no
 id | string | Used as a basic suffix for `id` attributes throughout the component. | |
 editable | boolean | Used as a flag for the component to be editable or not editable (without '+ New' button and column 'actions'). | `true` | no
+withDeleteConfirmation | boolean | Should user be asked to confirm delete action | `false` | no
+confirmationHeading | function/string/node | Confirmation modal heading. If it's a function - it will be called with id of the deleted item. | | yes
+confirmationMessage | string/node | Confirmation modal message | | yes
```

Comment:
I think the `confirmationHeading` and `confirmationMessage` props should not be marked as required.

Reply:
We don't have a requirement for generic messages that can be added. Our messages are Authority specific so I passed them as props and made the props required